dll/win32/shlwapi/ordinal.c

```diff
@@ -4131,6 +4131,14 @@ HRESULT WINAPI SHLoadRegUIStringW(HKEY hkey, LPCWSTR value, LPWSTR buf, DWORD si
     if(RegQueryValueExW(hkey, value, NULL, &type, (LPBYTE)buf, &sz) != ERROR_SUCCESS)
         return E_FAIL;
 
+#ifdef __REACTOS__
```

Comment:
When the PR gets ready, please squash all the kbswitch commits together, and please have a separate commit for shlwapi with a correct commit title and log summarizing what the change is about.

Reply:
https://github.com/reactos/reactos/pull/5084